server/tests/participant.landing.e2e.test.ts

```diff
@@ -1,55 +0,0 @@
-/**
- * Tests for route /api/v1/participant-user/*
```

Comment:
Renamed this file to `participant-user.e2e.test.ts` to match the route file name